compiler/test/suites/linking.re

```diff
@@ -16,4 +21,17 @@ describe("linking", ({test}) => {
     {|import List from "list"; print(List.map(n => n + 1, [1, 2, 3]))|},
     "[2, 3, 4]\n",
   );
+  // --wasi-polyfill
+  assertWasiPolyfillRun(
+    "test/input/wasiPolyfill.gr",
+    "wasi_polyfill",
+    {|print("foo")|},
+    "foo\nfoo\nfoo\n",
+  );
+  assertWasiPolyfillRun(
+    "test/input/wasiPolyfillNoop.gr",
+    "wasi_polyfill_noop",
+    {|print("foo")|},
+    "",
+  );
```

Comment:
I think we should add a test for the situation where you use a WASI function not defined in the polyfill.

Reply:
Will follow-up with a PR 👍

Reply:
Please create an issue @ospencer

Reply:
https://github.com/grain-lang/grain/issues/840